docs/restore-paas-postgres-database.md

```diff
@@ -0,0 +1,98 @@
+# How to restore database from point in time backup
+
+The below steps can be used to restore a database to its point in time backup.
+We can restore to any point from 5 minutes to 7 days ago, with a resolution of one second.
+
+## Adding the user to SpaceDeveloper role
+
+The user must be in the SpaceDeveloper role to perform any of the below actions.
+A user can be added to the SpaceDeveloper role by anyone in the SpaceManager role using the below command
+
+- CF CLI
+`cf set-space-role <firstname.lastname@digital.education.gov.uk|Google SSO UID> dfe <space-name> SpaceDeveloper`
+
+- Makefile
+
+`make <ENV> set-space-developer USER_ID=<user id obtained from $(cf target) command> `
+
+The username of the person being added to the role can be obtained by asking them to run `cf target`, this is usually their @digital.education.gov.uk email address unless they have enabled Google SSO in their GovUK PaaS account.
+
+## Restoring from a point in time backup
+
+### 1. Set the correct space context and stop any apps bound to the database service instance
```

Comment:
Is there an existing process for putting up a maintenance page?  Might be useful to link to that here

Reply:
No maintenance page for the API. /status will return error if any dependencies (postgres, Redis, CRM) are down